src/sagemaker_pytorch_serving_container/default_inference_handler.py

```diff
@@ -31,10 +43,18 @@ def default_model_fn(self, model_dir):
 
         Returns: A PyTorch model.
         """
-        raise NotImplementedError(textwrap.dedent("""
-        Please provide a model_fn implementation.
-        See documentation for model_fn at https://github.com/aws/sagemaker-python-sdk
-        """))
+        if os.getenv(INFERENCE_ACCELERATOR_PRESENT_ENV) == 'true':
+            default_model_filename = "model.pt"
```

Comment:
Can we declare this as a constant similar to: https://github.com/aws/sagemaker-mxnet-serving-container/blob/master/src/sagemaker_mxnet_serving_container/default_inference_handler.py#L26-L30

Reply:
moved "model.pt" to a constant variable at top

Reply:
please remove this variable assignment and reference the constant above.